### PR TITLE
Incorporated changes for color config issue #7

### PR DESCRIPTION
**Pull request by quantizationbit**
deleting old PR as well

***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-hdr10plus/pull/19.html" title="Last updated on Nov 16, 2021, 6:15 PM UTC (d032e0a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-hdr10plus/19/fbefb1f...d032e0a.html" title="Last updated on Nov 16, 2021, 6:15 PM UTC (d032e0a)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -150,6 +157,16 @@ For the purpose of this specification, the following acronyms apply:
 Use of HDR10+ with AV1 T.35 OBUs:
 =================================
 
+Color Configuration
+-------------------
+
+Streams suitable for incorporating HDR10+ metadata as described in this specification shall use the following values for the AV1 color config:
+* color_primaries = 9 (BT.2020)
```
Comment (cconcolato) [inline index.bs:164]: Should BT.2020 be treated the same way as BT-2100, with a normative reference? or should be BT-2100 use an informative reference?

File: index.bs [hunk 2]
```diff
@@ -162,7 +162,7 @@ Color Configuration
 
 Streams suitable for incorporating HDR10+ metadata as described in this specification shall use the following values for the AV1 color config:
 * color_primaries = 9 (BT.2020)
-* transfer_characteristics = 16 ([[!ST-2084]] / [[!BT-2100]])
+* transfer_characteristics = 16 ([[!ST-2084]] / [[BT-2100]])
```
Comment (cconcolato) [inline index.bs:165]: Should we add BT-2020 in the list of references? and used `[[` `]]` ?

Comment (quantizationbit) [reply index.bs:165]: I don't think it is required so I took the second option in your earlier comment to change 2100 to an informative comment.

Comment (cconcolato) [reply index.bs:165]: Why should we have a reference to BT-2100 and not a reference to BT-2020?

**Comment by cconcolato** [comment]
Note that PR Preview is failing because of `Couldn't find 'ST-2084' in bibliography data.`

**Review by cconcolato — APPROVED** [review]

**Pull request merged**